Source/ACE.Entity/AceObject.cs

```diff
@@ -296,7 +283,7 @@ public string Name
         /// </summary>
         public uint ItemType
         {
-            get { return GetIntProperty(PropertyInt.ItemType).Value; }
+            get { return IntProperties.Find(x => x.PropertyId == (uint)PropertyInt.ItemType).PropertyValue; }
```

Comment:
That was an unintentional change.  It should use the GetIntProperty interface, will change.